updates/35/updates.sql

```diff
@@ -298,5 +298,205 @@ INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0x2D189eABb
 *
-INSERT INTO assets(identifier, name, type) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS Token(PoS)', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'A', 137, '0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714637112, NULL);
+INSERT INTO assets(identifier, name, type) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS Token (PoS)', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'A', 137, '0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714637112, NULL);
 *
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0xDcEe70654261AF21C44c093C300eD3Bb97b78192', 'Wrapped OETH', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0xDcEe70654261AF21C44c093C300eD3Bb97b78192', 'A', 1, '0xDcEe70654261AF21C44c093C300eD3Bb97b78192', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0xDcEe70654261AF21C44c093C300eD3Bb97b78192', 'WOETH', 'wrapped-oeth', '', NULL, 1680301283, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:42161/erc20:0xD8724322f44E5c58D7A815F542036fb17DbbF839', 'Wrapped OETH', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:42161/erc20:0xD8724322f44E5c58D7A815F542036fb17DbbF839', 'A', 42161, '0xD8724322f44E5c58D7A815F542036fb17DbbF839', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:42161/erc20:0xD8724322f44E5c58D7A815F542036fb17DbbF839', 'WOETH', 'wrapped-oeth', '', NULL, NULL, NULL);
+*
```

Comment:
this one is missing the start date

Reply:
also seems to have a cryptocompare iD. https://www.cryptocompare.com/coins/woeth/overview Is not active but I would just add it